addons/payment_worldline/models/payment_transaction.py

```diff
@@ -184,7 +184,13 @@ def _send_payment_request(self):
         }
 
         # Make the payment request to Worldline.
-        response_content = self.provider_id._worldline_make_request('payments', payload=payload)
+        response_content = self.provider_id._worldline_make_request(
+            'payments',
+            payload=payload,
+            idempotency_key= payment_utils.generate_idempotency_key(
+                self, scope='payment_request_token_worldline'
```

Comment:
`_worldline_make_request` will never send a non-Wordline request, so we can safely simplify the scope:
```suggestion
                self, scope='payment_request_token'
```